javascript/javascript1/week2/candyhelper.js

```diff
@@ -0,0 +1,44 @@
+const boughtCandyPrices = [];
+const candyTpye = "Sweet";
+function addCandy(candyTpye, weight) {
```

Comment:
There is nothing wrong with this function and how you solve , but I would say it is not the best approach. For start I would sore candy types and prices in a variable. So you need to do a bit of research and which data type is best to use when you have multiple types and need to have sort of easy acces to them later on. 
Then you could have a function with two parameters 1.candy type 2. amount . and returns/push the price .

Reply:
A problem can be solved in many ways I thought as the teacher taught us switch cases so I can go with this approach. According to your suggestion, it can be solved with the help of json (dictionary) object.